src/rust/plugin-registry/src/db/client.rs

```diff
@@ -112,7 +112,8 @@ impl PluginRegistryDbClient {
                 id,
                 plugin_id,
                 deploy_time,
-                status AS "status: PluginDeploymentStatus"
+                status AS "status: PluginDeploymentStatus",
+                deployed
```

Comment:
PluginDeploymentStatus not containing 'is_deployed' is not ideal, but whatever it's fine for now :)